src/app/page.tsx

```diff
@@ -2,90 +2,207 @@ import React from 'react';
+  },
+  {
+    name: 'Docker',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Open-Source',
+    projects: ['OpenTierBoy'],
+  },
+  {
+    name: 'Enterprise',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Expo (React Native)',
+    projects: [],
+  },
+  {
+    name: 'Supabase',
+    projects: [],
+  },
+  {
+    name: 'REST',
+    projects: ['Levain'],
+  },
+  {
+    name: 'GraphQL',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Jest',
+    projects: ['OpenTierBoy', 'Levain'],
+  },
+  {
+    name: 'Mocha',
+    projects: [],
+  },
+  {
+    name: 'Cypress',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Playwright',
+    projects: ['Levain'],
+  },
+  {
+    name: 'SQL',
+    projects: ['Levain'],
+  },
+  {
+    name: 'PostgreSQL',
+    projects: ['Levain'],
+  },
+  {
+    name: 'TypeORM',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Sequelize',
+    projects: ['Levain'],
+  },
+  {
+    name: 'KnexJS',
+    projects: [],
+  },
+  {
+    name: 'Redis',
+    projects: [],
+  },
+  {
+    name: 'NestJS',
+    projects: ['Levain'],
+  },
+  {
+    name: 'BunJS',
+    projects: [],
+  },
+  {
+    name: 'Vite',
+    projects: [],
+  },
+  {
+    name: 'HTML',
+    projects: ['OpenTierBoy', 'Levain'],
+  },
+  {
+    name: 'Material UI',
+    projects: [],
+  },
+  {
+    name: 'Tailwind CSS',
+    projects: ['OpenTierBoy', 'Levain'],
+  },
+  {
+    name: 'Shadcn UI',
+    projects: ['OpenTierBoy'],
+  },
+  {
+    name: 'Blockchain',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Web 3',
+    projects: ['Levain'],
+  },
+  {
+    name: 'UTXO',
+    projects: ['Levain'],
+  },
+  {
+    name: 'EVM',
+    projects: ['Levain'],
+  },
+  {
+    name: 'SVM',
+    projects: ['Levain'],
+  },
+  {
+    name: 'TVM',
+    projects: ['Levain'],
+  },
+  {
+    name: 'Unreal Engine',
+    projects: [],
   },
-};
-
-const TECHNOLOGIES: { name: string }[] = [
-  { name: 'React' },
-  { name: 'NextJS' },
-  { name: 'NestJS' },
-  { name: 'Typescript' },
-  { name: 'Javascript' },
-  { name: 'Expo (React Native)' },
-  { name: 'Supabase' },
-  { name: 'REST' },
-  { name: 'GraphQL' },
-  { name: 'Jest' },
-  { name: 'Mocha' },
-  { name: 'Cypress' },
-  { name: 'Playwright' },
-  { name: 'SQL' },
-  { name: 'PostgreSQL' },
-  { name: 'TypeORM' },
-  { name: 'Sequelize' },
-  { name: 'KnexJS' },
-  { name: 'Redis' },
-  { name: 'NodeJS' },
-  { name: 'BunJS' },
-  { name: 'Vite' },
-  { name: 'HTML' },
-  { name: 'Material UI' },
-  { name: 'Tailwind CSS' },
-  { name: 'Blockchain' },
-  { name: 'Web 3' },
-  { name: 'UTXO' },
-  { name: 'EVM' },
-  { name: 'SVM' },
-  { name: 'TVM' },
-  { name: 'Unreal Engine' },
 ];
 
 // Function to calculate font size based on index
 const getTechnologiesFontSize = (index: number) => {
-  if (index < 3) return 'text-lg'; // Top 3 technologies
-  if (index < 7) return 'text-base';
+  if (index < 3) return 'text-xs'; // Top 3 technologies
+  if (index < 7) return 'text-xs';
```

Comment:
Leave this here intentionally in case I wanna reuse this approach in the future.